CONTRIBUTING.md

```diff
@@ -75,7 +75,27 @@ root directory.
 In case you want to debug with remote JVM, you can do it with this command:
 `./mvnw spring-boot:run -Dspring-boot.run.jvmArguments="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005"`
 
-Note: TODO add here about database and running HOW-TOs.
+Note: You need a running Postgres service to have a fully working environment. You
+can do that with Docker or Docker Compose. Follow these steps
+
+With Docker, build and run:
+
+```shell
+cd database && docker build -t bcgov/nr-spar-backend-database:snapshot . && cd ..
+
+docker run -t -i --net=host \
```

Comment:
You don't need to set the network to use a host configuration, just expose the ports with `-p 5432:5432`.

Reply:
I changed to host due to be unable to connect to the service only with the port. I received `Connection refused` from localhost.